packages/tools/pluggable-widgets-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/pluggable-widgets-tools",
-  "version": "9.0.2",
+  "version": "9.0.3",
```

Comment:
Does it make sense to add a changelog for this too?

Reply:
It does. We will do so in PR #582.